python/lsst/meas/base/baseMeasurement.py

```diff
@@ -433,6 +422,46 @@ def doMeasurement(self, plugin, measRecord, *args, **kwds):
                 plugin.name, measRecord.getId(), error)
             plugin.fail(measRecord)
 
+
+class BaseMeasurementTask(SimpleBaseMeasurementTask):
+    """Ultimate base class for all measurement tasks.
+
+    Parameters
+    ----------
+    algMetadata : `lsst.daf.base.PropertyList` or `None`
+        Will be modified in-place to contain metadata about the plugins being
+        run. If `None`, an empty `~lsst.daf.base.PropertyList` will be
+        created.
+    **kwds
+        Additional arguments passed to `lsst.pipe.base.Task.__init__`.
+
+    Notes
+    -----
+    This base class for `SingleFrameMeasurementTask` and
+    `ForcedMeasurementTask` mostly exists to share code between the two, and
+    generally should not be used directly.
```

Comment:
This note is repeated, and again I'm at a loss as to the difference between the two.

Reply:
I'm clarifying in the docstrings now.